LICENSE.md

```diff
@@ -0,0 +1,206 @@
+
+* Copyright 2020 Balena Ltd. under the Apache 2.0 License included in this file.
+* Copyright 2018 Zeit Inc. under the MIT License included in this file.
```

Comment:
This "multiple licensing" Apache 2.0 + MIT was discussed in the architecture call of 23 Apr 2020. It satisfies the legal requirements of the original repo (https://github.com/zeit/dockerignore/blob/0.0.5/license.md) and balena's own licensing policy (Apache 2.0) and balena CI checks.